widgets/heatmap/options.go

```diff
@@ -59,6 +59,13 @@ func (o option) set(opts *options) {
 	o(opts)
 }
 
+// CellWidth set the width of each cell.
```

Comment:
It may be worth to spend a bit more time here, feels like this is an important detail about how HeatMap works / how it assigns data to cells, but the current description of this option doesn't really help me to understand it. A width of a terminal cell is fixed to just 1. What does it mean when we change it to a higher number? What impact does it have on how we are going to present the data?

Reply:
The cellWidth is the width of cells(or grids) in the heat map, not the terminal cell.
The default height of each cell (grid) is 1 and the width is 3.
In the figure below, the cell is framed in red：

<img width="799" alt="Jietu20201122-230412" src="https://user-images.githubusercontent.com/26627380/99907322-2731b980-2d17-11eb-9c3c-02ae35821803.png">

Reply:
Thank you, I understand this now. Would you be able to update the comment on the option, so that it explains this for our users?

Reply:
OK